src/app/reducers/api-response.ts

```diff
@@ -87,12 +87,18 @@ export function reducer(state: State = initialState, action: api.Actions | pagin
 			let hashtags = Array.from(new Set(tagStrings)).map(tag => {
 					return { tag, count: tagStrings.filter(y => y === tag).length };
 			});
+			let oldapiaggregations = state.aggregations;
+			let newapiaggregations = apiResponse.aggregations;
+			if(newapiaggregations === null)
```

Comment:
Add space between the `if` syntax and the parenthesis and Please make the bracket on the same line as the if statement.
Basically, Make your code consistent with the project.